model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -169,29 +169,38 @@ def make_model_config(param_combo, config, client, gpus, model,
             dict of model properties
         model_repository: str
             path to the model repository on the file system
-        variant_name_manager: ModelVariantNameManager
+        model_variant_name_manager: ModelVariantNameManager
         """
         model_name = model.model_name()
-        variant_name = variant_name_manager.get_model_variant_name(
-            model_name, param_combo)
 
         model_config_dict = BaseModelConfigGenerator.get_base_model_config_dict(
             config, client, gpus, model_repository, model_name)
 
-        model_config_dict['name'] = variant_name
-        logger.info("")
-        logger.info(f"Creating model config: {model_config_dict['name']}")
-
+        logger_str = []
         if param_combo is not None:
             for key, value in param_combo.items():
                 if value is not None:
                     BaseModelConfigGenerator._apply_value_to_dict(
                         key, value, model_config_dict)
 
                     if value == {}:
-                        logger.info(f"  Enabling {key}")
+                        logger_str.append(f"  Enabling {key}")
                     else:
-                        logger.info(f"  Setting {key} to {value}")
+                        logger_str.append(f"  Setting {key} to {value}")
+
+        (variant_found,
+         variant_name) = model_variant_name_manager.get_model_variant_name(
+             model_name, model_config_dict, param_combo)
+
+        model_config_dict['name'] = variant_name
+        logger.info("")
+        if variant_found:
+            logger.info(
+                f"Found existing model config: {model_config_dict['name']}")
+        else:
+            logger.info(f"Creating model config: {model_config_dict['name']}")
+        for str in logger_str:
+            logger.info(str)
```

Comment:
Is there a reason why we are logging in this manner?

Reply:
Yes, we now create the param combo before the variant name, but the logger print order is goes variant name then param combo.